### PR TITLE
components: card-grid-course

### LINKED ISSUE #37: [Request] card-grid-course
https://dribbble.com/shots/9323134-Web-Category-Exploration/attachments/1367507?mode=media

image

### PR DESCRIPTION
**Pull request by fResult**
closes #37

### CODE REVIEW

**Comment by siriwatknp** [comment]
| Results |  |
| --- | --- |
| Build Results | https://www.chromatic.com/build?appId=64e95d8f1f548945b1225cb8&number=61 |
| Storybook Preview | https://64e95d8f1f548945b1225cb8-ykbyjaxiyd.chromatic.com/ |
| Component Count | 53 |

**Comment by fResult** [comment]
[Issue #37](https://github.com/siriwatknp/joy-treasury/issues/37) krub

**Comment by siriwatknp** [comment]
| Results |  |
| --- | --- |
| Build Results | https://www.chromatic.com/build?appId=64e95d8f1f548945b1225cb8&number=64 |
| Storybook Preview | https://64e95d8f1f548945b1225cb8-cskkiuxjvr.chromatic.com/ |
| Component Count | 53 |

File: blocks/card-grid-course/src/CardGridCourse.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import CardGridCourse, { CardGridCourseProps, COLORS } from "./CardGridCourse";
+import IconScience from "./IconScience";
+import IconWrapper from "./IconWrapper";
+
+const meta: Meta<typeof CardGridCourse> = {
+  title: "Card/Grid/Course",
+  component: CardGridCourse,
+  parameters: {
+    layout: "centered",
+    githubUsername: "", // (optional) Your github username. If provided, your avatar will be displayed in the story toolbar
```
Comment (siriwatknp) [inline blocks/card-grid-course/src/CardGridCourse.stories.tsx:15]:
```suggestion
    githubUsername: "fResult", // (optional) Your github username. If provided, your avatar will be displayed in the story toolbar
```

File: blocks/card-grid-course/src/CardGridCourse.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  url = "#",
+}) => {
+  return (
+    <Card
+      variant="solid"
+      sx={{
+        transition: TRANSITION_ALL,
+        bgcolor: COLORS.white,
+        "&:focus-within": {
+          transform: "scale(1.01)",
```
Comment (siriwatknp) [inline blocks/card-grid-course/src/CardGridCourse.tsx:36]:
```suggestion
          transform: "scale(1.05)",
```

**Review by siriwatknp — APPROVED** [review]
👍 Great first PR! looking forward to the next one.

**Comment by siriwatknp** [comment]
@all-contributors please add @fResult for code.

**Comment by siriwatknp** [comment]
| Results |  |
| --- | --- |
| Build Results | https://www.chromatic.com/build?appId=64e95d8f1f548945b1225cb8&number=71 |
| Storybook Preview | https://64e95d8f1f548945b1225cb8-vfihpqvghz.chromatic.com/ |
| Component Count | 55 |

**Pull request merged**